File: PizzaShop.EntityFramework/Entities/Pizza.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace PizzaShop.EntityFramework.Entities
{
    public class Pizza : IEntity
    {
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public string Description { get; set; }

        public string ImgPath { get; set; }

        public virtual ICollection<PizzaIngredient> PizzaIngredients { get; set; }
        public virtual ICollection<Price> Prices { get; set; }
    }
}

File: PizzaShop/ClientApp/src/app/models/pizza.ts
import { PizzaPrice } from './pizza-price';
import { PizzaIngredient } from './pizza-ingredient';

export class Pizza {
    public id: number;
    public name: string;
    public description: string;
    public imgPath: string;
    public prices: PizzaPrice[];
    public ingredients: PizzaIngredient[];
    public image?: ArrayBuffer;
}
